en/docs/learn/account-locking-by-failed-login-attempts.md

```diff
@@ -210,4 +210,137 @@ failed login attempts. To configure this, follow the steps below.
         can edit and customize the email template. For more information on
         how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails).
+
+#### Configuring email templates for WUM Updated version
+
+!!! warning
+    Make sure you have a WUM updated WSO2 Identity Server 5.9.0 pack. If not this section is not relevant.
+    For more information on how to WUM update, see [Updating WSO2 Products](../../administer/getting-wso2-updates)
     
+WUM Updated WSO2 Identity Server 5.9.0 requires separate email templates for notifying, 
+
+- Account locking by exceeding `Maximum Failed Login Attempts`
+- Account unlocking by exceeding `Account Unlock Time`
+
+Please add the following email templates by referring to the instructions in 
```

Comment:
Generally we avoid using the word "please" in our docs to keep the tone neutral. We can start this sentence with "Add the following..."

Reply:
done